projects/common/src/telemetry/user-telemetry-internal.service.ts

```diff
@@ -0,0 +1,155 @@
+import { Injectable, Injector, OnDestroy } from '@angular/core';
+import { NavigationEnd, Router } from '@angular/router';
+import { Observable, ReplaySubject, Subject } from 'rxjs';
+import { filter, map, takeUntil } from 'rxjs/operators';
+import { Dictionary } from '../utilities/types/types';
+import { UserTelemetryProvider, UserTelemetryRegistrationConfig, UserTraits } from './telemetry';
+
+@Injectable({ providedIn: 'root' })
+export class UserTelemetryInternalService implements OnDestroy {
```

Comment:
there is no destroy for a root injectable, implementing ondestroy doesn't do anything since this lives the whole life of the app.

Reply:
okay. Then I can remove the `takeUntil` logic as well.